frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/types.ts

```diff
@@ -0,0 +1,4 @@
+export type CephHealthCheck = {
```

Comment:
```suggestion
type CephHealthCheck = {
```
Move it to status card if it is not being used anywhere else.